galasa-extensions-parent/dev.galasa.auth.couchdb/src/main/java/dev/galasa/auth/couchdb/internal/CouchdbAuthStore.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package dev.galasa.auth.couchdb.internal;
+
+import static dev.galasa.extensions.common.Errors.*;
+
+import java.io.IOException;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.logging.Log;
+
+import dev.galasa.extensions.common.api.HttpClientFactory;
+import dev.galasa.extensions.common.api.LogFactory;
+import dev.galasa.extensions.common.couchdb.CouchdbException;
+import dev.galasa.extensions.common.couchdb.CouchdbStore;
+import dev.galasa.extensions.common.couchdb.CouchdbValidator;
+import dev.galasa.extensions.common.couchdb.pojos.ViewRow;
+import dev.galasa.extensions.common.api.HttpRequestFactory;
+import dev.galasa.framework.spi.auth.IAuthToken;
+import dev.galasa.framework.spi.auth.IAuthStore;
+import dev.galasa.framework.spi.auth.User;
+import dev.galasa.framework.spi.utils.ITimeService;
+import dev.galasa.framework.spi.auth.AuthStoreException;
+
+/**
+ * When CouchDB is being used to store user-related information, including authentication
+ * tokens, this class is called upon to implement the auth store. This class registers the
```

Comment:
we don't store auth tokens, but we do store information about tokens, enough so we can list and delete them.

Reply:
Clarified